openproblems/data/lung.py

```diff
@@ -0,0 +1,45 @@
+from . import utils
+
+import os
+import scprep
+import tempfile
+
+# sparsified from https://figshare.com/articles/dataset/Benchmarking_atlas-level_data_integration_in_single-cell_genomics_-_integration_task_datasets_Immune_and_pancreas_/12420968/2 # noqa: E501
+URL = "https://figshare.com/ndownloader/files/24539942"
+
+
+@utils.loader(data_url=URL, data_reference="luecken2022benchmarking")
+def load_lung(test=False):
+    """Download lung data from figshare."""
+    import scanpy as sc
+
+    if test:
+        # load full data first, cached if available
+        adata = load_lung(test=False)
+
+        # Subsample immune data to two batches with 250 cells each
+        adata = adata[:, :500].copy()
+        batch1 = adata[adata.obs.dataset == "10x_Biopsy"][:250]
+        batch2 = adata[adata.obs.dataset == "10x_Transplant"][:250]
+        adata = batch1.concatenate(batch2)
+        # Note: could also use 200-500 HVGs rather than 200 random genes
```

Comment:
again, do we need to check cell type numbers here? I guess we need 50 per cell type now as otherwise some metrics will fail given new settings

Reply:
selected two batches with 3 cts with >50 cells now